src/behaviors/Visibility/Visibility.js

```diff
@@ -20,6 +20,9 @@ export default class Visibility extends Component {
     /** Primary content. */
     children: PropTypes.node,
 
+    /** The scroll context visibility should use. */
+    context: PropTypes.object,
```

Comment:
There is a default value of `window`. `isRequired` means that the user must specify the value, right?